docs/_includes/component/footer.njk

```diff
@@ -15,15 +15,20 @@
 
   <footer class="main-footer">
     <img style="max-width: 135px;" src="https://static.redhat.com/libs/redhat/brand-assets/2/corp/logo--on-dark.svg" alt="Red Hat" />
-    <p>Copyright &copy;<script type="text/javascript">document.write(new Date().getFullYear());</script> Red Hat, Inc.</p>
+    <p>Copyright &copy; 2021 Red Hat, Inc.</p>
```

Comment:
Can we update this year per this same PR?

Reply:
yes, but isn't it [better to declare an older copyright](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated)?

Reply:
@markcaron suggested "2021-2022"